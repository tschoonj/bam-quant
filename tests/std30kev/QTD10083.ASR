File: C:\xrfdaten\2014KW30\chris\std30kev\std10083.spe
Start: 22.07.2014 10:31:43
Stop: 22.07.2014 10:33:23
BESSY: 298.9384478 298.6911369
IONI: 1.960423E-11 1.959677E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 95474
Run: 100000
Dead: 5
ICR: 8680
OCR: 0
Accumulated BESSY: 59389.7162541
Accumulated Ioni: 3.86742268E-09
N Accumulation: 199
Normfactor:  5.3894732e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Bi
exp_x 152.720397949219 152.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  83   2     10.828    119300.       332.     105.76
