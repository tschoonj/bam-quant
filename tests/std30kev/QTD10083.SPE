$DATA:
0 2000
22072
14095
5988
1786
366
47
6
0
0
0
0
0
0
0
0
0
41
218
227
236
226
224
230
218
209
217
193
216
200
166
181
165
166
170
172
166
140
153
139
146
124
128
123
131
120
102
124
119
115
106
98
107
111
97
82
107
87
84
119
85
91
80
93
91
92
98
87
72
92
95
66
95
99
86
98
87
69
86
72
81
64
67
55
74
75
91
82
72
87
57
66
80
67
53
68
66
74
66
69
76
79
64
69
70
71
69
52
83
77
62
63
50
62
63
63
54
59
61
59
83
59
47
65
67
69
64
62
53
56
76
48
64
54
56
65
70
57
41
55
50
62
47
54
57
47
50
42
49
58
64
60
54
43
44
49
47
57
47
50
48
49
40
67
40
61
55
44
47
41
48
65
56
63
48
40
51
43
54
56
47
43
56
34
45
48
60
43
54
41
39
42
40
49
49
38
39
52
41
45
46
42
50
49
46
51
28
50
49
46
50
36
56
47
52
33
45
36
43
44
45
34
45
41
39
37
44
38
49
38
47
43
36
40
38
39
48
40
38
43
49
40
34
51
40
30
46
49
42
39
50
37
39
34
38
36
35
35
45
52
42
33
33
38
40
43
51
39
53
50
29
45
46
29
38
44
43
51
42
37
21
36
34
38
31
44
27
29
24
36
43
43
34
43
35
53
37
43
30
30
40
33
56
48
32
39
35
42
42
39
34
38
37
49
59
68
68
67
83
59
59
59
54
50
43
38
52
40
32
41
43
36
34
40
25
33
41
39
44
46
35
43
43
45
34
39
40
46
39
46
45
48
41
35
41
28
40
37
35
36
35
33
35
48
30
48
32
27
35
37
49
46
39
39
42
48
44
51
45
33
35
39
38
39
38
39
35
47
39
44
33
34
34
32
49
29
37
46
39
45
37
43
44
45
47
46
43
37
36
29
37
37
32
46
39
45
45
43
49
37
47
41
33
44
51
40
55
48
45
35
46
46
51
42
53
47
53
46
46
53
53
58
43
57
55
54
52
52
59
75
76
51
71
45
67
54
61
64
66
88
77
82
117
104
139
159
182
176
182
191
162
156
139
111
75
78
75
68
53
49
74
56
63
62
57
66
59
54
63
62
80
64
52
54
64
60
68
56
78
77
84
76
73
72
64
82
83
89
95
96
74
90
91
109
108
88
93
93
91
120
143
152
149
193
282
379
557
819
1239
1837
2621
3629
5096
6639
8286
10047
11302
12190
12382
11673
10443
8747
6639
5127
3401
2358
1463
908
544
363
254
186
156
155
141
121
151
140
139
141
130
127
133
133
116
130
115
112
127
123
119
136
136
171
238
227
321
401
478
490
508
518
552
506
439
349
333
264
217
175
152
173
131
161
152
145
132
142
164
171
151
169
183
198
150
186
185
208
191
205
271
283
349
407
478
577
657
741
854
1052
1064
1282
1486
1666
1984
2387
2791
3153
3411
3614
3730
3563
3358
3276
3343
3946
4929
6920
9468
13606
18465
24150
29682
35290
39161
41531
41680
39839
35600
30567
25370
20667
15952
12393
10045
8043
6710
5630
4790
4064
3381
3055
2639
2312
2171
1857
1738
1431
1261
998
805
646
483
356
292
241
218
208
199
218
205
229
233
242
226
224
168
188
169
131
114
98
95
76
65
50
71
61
46
45
69
48
59
45
57
71
51
46
61
61
60
77
78
65
60
69
70
61
76
83
83
86
83
121
94
109
166
186
215
210
241
279
273
271
240
261
265
210
170
195
174
172
208
237
331
487
623
955
1364
1940
2623
3358
4262
5125
5591
6009
6156
6022
5497
4919
4182
3289
2645
2054
1503
1187
1068
986
1084
1243
1436
1690
1872
2148
2358
2566
2597
2607
2574
2463
2207
1853
1605
1296
986
762
543
391
298
224
182
121
91
107
86
62
73
79
94
101
153
184
187
284
290
339
390
406
370
344
330
313
260
213
190
115
87
65
41
29
24
27
18
17
22
12
11
22
16
17
11
12
13
9
10
2
10
12
10
9
13
7
8
13
10
10
8
9
15
8
9
4
11
6
7
8
14
9
9
11
7
8
9
5
17
5
8
12
12
11
19
13
8
20
14
18
12
14
14
9
21
10
18
16
15
10
13
8
15
5
8
14
13
8
14
10
9
10
11
8
7
7
5
8
8
13
13
10
9
14
5
10
2
8
8
12
7
10
7
15
9
6
9
8
11
11
7
7
9
12
8
11
10
7
5
5
7
3
9
9
3
9
7
5
10
7
6
7
6
13
6
9
4
8
6
6
2
7
6
8
7
4
7
3
15
9
8
5
4
3
4
9
6
8
10
5
6
5
5
5
9
4
5
2
9
5
4
5
9
2
3
7
7
6
4
5
6
10
9
6
4
6
4
7
3
5
5
4
4
2
5
4
6
7
6
5
7
6
4
4
8
8
2
8
4
3
2
7
5
6
7
3
11
6
4
4
9
4
12
3
4
6
0
6
9
5
6
7
6
11
4
3
2
4
5
5
6
4
4
3
2
6
4
7
8
3
4
3
6
6
5
4
6
5
2
6
5
6
3
9
4
3
6
5
2
4
2
3
5
5
5
1
7
5
8
5
4
4
4
3
5
7
5
3
2
1
3
1
3
2
9
1
0
1
8
5
5
5
2
2
2
4
4
3
6
2
3
2
2
4
3
2
7
5
0
2
1
7
2
3
4
1
2
2
4
2
4
1
5
3
5
5
6
7
5
3
5
8
6
3
3
3
4
3
1
1
2
5
4
5
7
8
7
7
7
11
8
14
8
6
10
12
11
8
12
7
10
7
9
2
5
7
3
3
6
2
4
3
7
1
1
4
4
2
2
4
3
1
2
2
3
3
5
1
1
2
2
4
1
7
2
1
2
4
5
3
2
2
5
4
0
3
2
5
3
5
5
8
8
8
6
13
7
9
13
15
7
12
13
8
8
8
11
7
9
9
9
9
3
7
5
2
4
4
2
7
6
6
3
1
5
8
5
8
5
6
9
6
9
3
12
11
9
12
14
13
14
17
13
20
16
17
14
21
14
8
9
13
8
5
13
11
7
10
5
11
8
6
10
3
5
6
6
10
4
4
5
4
1
2
2
2
2
2
2
2
3
2
5
2
3
1
2
2
1
0
2
1
2
4
1
0
1
5
2
5
2
1
4
4
2
4
5
6
0
3
7
3
6
7
4
5
6
2
5
4
2
7
5
3
2
3
5
2
3
4
2
4
1
4
1
6
2
4
3
1
3
1
5
3
3
5
5
4
3
6
6
6
12
2
6
7
3
9
6
3
7
5
3
5
5
7
5
9
4
5
10
3
2
3
7
7
0
2
4
1
5
3
4
6
2
1
4
5
6
3
0
3
3
6
1
6
2
4
1
4
7
2
3
2
2
4
4
6
4
6
4
2
3
1
2
1
1
1
2
3
2
1
1
6
5
2
3
0
2
1
1
0
3
2
5
8
5
6
2
1
2
2
3
4
3
3
3
2
1
7
3
8
1
4
4
5
3
2
5
2
1
4
1
1
4
6
4
1
7
2
4
3
3
2
3
4
1
7
3
7
4
1
5
6
3
7
3
2
1
4
6
5
4
6
3
0
9
6
3
4
2
7
7
3
8
5
2
7
9
1
6
5
9
7
10
5
6
8
6
3
4
6
1
3
7
4
7
6
5
7
6
8
9
6
5
4
2
6
5
4
5
3
4
2
7
11
3
5
6
5
5
4
1
3
9
3
7
6
3
5
9
8
5
6
6
6
6
8
7
8
3
7
5
10
4
4
7
8
13
7
9
5
9
10
5
10
8
14
5
4
15
7
5
9
2
7
10
8
7
10
7
10
10
8
8
9
8
4
10
17
12
7
8
9
16
9
15
9
8
10
12
13
11
8
9
8
11
12
12
12
11
9
12
12
16
10
11
10
16
15
13
14
17
20
15
16
14
11
17
19
10
15
14
9
15
15
13
14
8
14
13
13
10
18
10
20
14
19
19
24
20
23
29
21
31
41
31
49
54
53
62
66
76
63
72
83
92
77
84
96
71
84
85
95
75
80
86
78
83
81
105
80
69
69
65
66
53
60
60
50
47
45
40
25
21
38
25
27
17
10
11
11
7
7
5
2
1
1
3
3
0
3
1
1
2
2
1
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10083.spe
Start: 22.07.2014 10:31:43
Stop: 22.07.2014 10:33:23
BESSY: 298.9384478 298.6911369
IONI: 1.960423E-11 1.959677E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 95474
Run: 100000
Dead: 5
ICR: 8680
OCR: 0
Accumulated BESSY: 59389.7162541
Accumulated Ioni: 3.86742268E-09
N Accumulation: 199
Normfactor:  5.3894732e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Bi
 
exp_x 152.720397949219 152.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  83   2     10.828    119300.       332.     105.76
