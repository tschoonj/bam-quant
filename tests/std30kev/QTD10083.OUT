     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:29:03  บ   
     บ  Spectrum: QTD10083.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        3 iterations done    บ   
     บ  ChiSquare =    27.2   last change =   .00%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -35.3 ๑   100.0   บ     -35.3 ๑     1.0   บ   
     บ  GAIN (eV/ch)  บ     20.223 ๑   2.000   บ    20.223 ๑    .002   บ   
     บ det NOISE (eV) บ      160.8 ๑    40.0   บ     160.8 ๑     3.5   บ   
     บ  FANO factor   บ       .069 ๑    .050   บ      .069 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Bi-L1                            123230. ๑    458.           บ   
     บ    L1M3       13.210    .37289       45951. ๑    172.    174.3  บ   
     บ              654.973    212.94         269.           9.86E-01  บ   
     บ    L1M2       12.691    .34774       42852. ๑    160.     36.8  บ   
     บ              629.308    211.14         288.           9.85E-01  บ   
     บ    L1N3       15.710    .10937       13477. ๑     52.      4.3  บ   
     บ              778.596    221.42         209.           9.88E-01  บ   
     บ    L1N2       15.582    .09174       11305. ๑     44.     13.3  บ   
     บ              772.267    221.00         212.           9.88E-01  บ   
     บ    L1O3       16.295    .02172        2677. ๑     17.      5.0  บ   
     บ              807.524    223.36         192.           9.87E-01  บ   
     บ    L1O2       16.271    .01882        2319. ๑     16.      5.0  บ   
     บ              806.337    223.28         192.           9.87E-01  บ   
     บ    L1M5       13.808    .01950        2402. ๑     19.     21.2  บ   
     บ              684.543    215.00         274.           9.86E-01  บ   
     บ    L1M4       13.700    .01302        1604. ๑     18.    125.4  บ   
     บ              679.203    214.63         278.           9.88E-01  บ   
     บ    L1N5       15.951    .00320         395. ๑     14.      3.2  บ   
     บ              790.514    222.23         201.           9.87E-01  บ   
     บ    L1N4       15.904    .00200         247. ๑     14.      3.2  บ   
     บ              788.190    222.07         203.           9.88E-01  บ   
     บ                                                                 บ   
     บ  2 Bi-L2                            503550. ๑    783.           บ   
     บ    L2M4       13.024    .77725      391383. ๑    609.    206.1  บ   
     บ              645.775    212.30         275.           9.86E-01  บ   
     บ    L2N4       15.248    .16913       85165. ๑    133.    175.8  บ   
     บ              755.751    219.88         223.           9.87E-01  บ   
     บ    L2M1       11.712    .02148       10817. ๑     25.    214.0  บ   
     บ              580.897    207.69         329.           9.85E-01  บ   
     บ    L2N1       14.773    .00560        2822. ๑     16.     17.2  บ   
     บ              732.262    218.28         239.           9.87E-01  บ   
     บ    L2O4       15.685    .02444       12305. ๑     24.      4.7  บ   
     บ              777.360    221.34         209.           9.88E-01  บ   
     บ    L2O1       15.551    .00120         605. ๑     15.     25.8  บ   
     บ              770.734    220.89         214.           9.88E-01  บ   
     บ    L2M3       12.534    .00090         452. ๑     17.    181.4  บ   
     บ              621.545    210.59         294.           9.84E-01  บ   
     บ                                                                 บ   
     บ  3 Bi-L3                            155322. ๑    433.           บ   
     บ    L3M5       10.839    .68965      107118. ๑    299.    105.8  บ   
     บ              537.728    204.57         370.           9.82E-01  บ   
     บ    L3M4       10.731    .07843       12182. ๑     39.     96.2  บ   
     บ              532.388    204.18         375.           9.81E-01  บ   
     บ    L3N5       12.980    .14081       21871. ๑     63.    198.0  บ   
     บ              643.599    212.14         277.           9.86E-01  บ   
     บ    L3N4       12.955    .01576        2447. ๑     18.    198.5  บ   
     บ              642.363    212.06         278.           9.86E-01  บ   
     บ    L3M1        9.420    .04081        6339. ๑     27.    559.0  บ   
     บ              467.559    199.39         443.           9.73E-01  บ   
     บ    L3N1       12.482    .01032        1603. ๑     18.    210.1  บ   
     บ              618.974    210.41         297.           9.84E-01  บ   
     บ    L3O1       13.259    .00221         343. ๑     16.    226.5  บ   
     บ              657.396    213.11         266.           9.86E-01  บ   
     บ    L3O5       13.395    .02201        3418. ๑     19.    465.3  บ   
     บ              664.121    213.58         261.           9.87E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                             409. ๑     30.           บ   
     บ    L2M1       26.043    .28703         118. ๑     12.      1.3  บ   
     บ             1289.535    253.49          62.           7.67E-01  บ   
     บ    L2N1       23.860    .14845          61. ๑     10.      1.1  บ   
     บ             1181.623    247.07          76.           8.44E-01  บ   
     บ    L3M5       28.269    .11758          48. ๑      8.      1.1  บ   
     บ             1399.645    259.89          53.           6.87E-01  บ   
     บ    L3N5       26.232    .06400          26. ๑      8.      1.6  บ   
     บ             1298.885    254.04          61.           7.60E-01  บ   
     บ    L3M1       25.712    .05924          24. ๑      8.      1.6  บ   
     บ             1273.203    252.53          64.           7.79E-01  บ   
     บ    L3N1       28.719    .03562          15. ๑      7.      2.1  บ   
     บ             1421.901    261.16          52.           6.72E-01  บ   
     บ               26.087    .03041          13. ๑      8.      1.5  บ   
     บ             1291.733    253.62          62.           7.65E-01  บ   
     บ               21.678    .01919           8. ๑     10.       .8  บ   
     บ             1073.711    240.47          98.           9.12E-01  บ   
     บ               23.752    .01688           7. ๑      9.      1.3  บ   
     บ             1176.282    246.75          77.           8.48E-01  บ   
     บ               24.049    .01655           7. ๑      9.      1.6  บ   
     บ             1190.973    247.63          75.           8.38E-01  บ   
     บ               28.602    .01646           7. ๑      7.      2.0  บ   
     บ             1416.084    260.83          52.           6.76E-01  บ   
     บ               23.530    .01532           6. ๑      9.      1.7  บ   
     บ             1165.291    246.08          79.           8.55E-01  บ   
     บ               24.733    .01493           6. ๑      8.      2.4  บ   
     บ             1224.792    249.66          70.           8.14E-01  บ   
     บ               28.458    .01311           5. ๑      7.      1.5  บ   
     บ             1408.996    260.42          52.           6.81E-01  บ   
     บ               27.939    .01213           5. ๑      7.      1.1  บ   
     บ             1383.314    258.95          54.           6.99E-01  บ   
     บ               30.496    .01204           5. ๑      7.      1.2  บ   
     บ             1509.756    266.13          49.           6.13E-01  บ   
     บ               26.537    .00921           4. ๑      8.      1.4  บ   
     บ             1313.989    254.93          60.           7.49E-01  บ   
     บ               22.441    .00878           4. ๑      9.      4.2  บ   
     บ             1111.454    242.80          89.           8.90E-01  บ   
     บ               30.946    .00730           3. ๑      7.      1.9  บ   
     บ             1532.012    267.37          48.           5.99E-01  บ   
     บ               29.305    .00691           3. ๑      7.      1.1  บ   
     บ             1450.868    262.81          54.           6.52E-01  บ   
     บ               25.901    .00660           3. ๑      8.      1.2  บ   
     บ             1282.554    253.08          63.           7.72E-01  บ   
     บ               26.416    .00474           2. ๑      8.      1.7  บ   
     บ             1308.015    254.58          60.           7.54E-01  บ   
     บ               21.570    .00437           2. ๑     10.       .8  บ   
     บ             1068.370    240.14         100.           9.15E-01  บ   
     บ               26.419    .00426           2. ๑      8.      1.7  บ   
     บ             1308.172    254.59          60.           7.53E-01  บ   
     บ               28.908    .00397           2. ๑      7.      2.3  บ   
     บ             1431.252    261.69          51.           6.65E-01  บ   
     บ               27.794    .00390           2. ๑      7.      1.0  บ   
     บ             1376.157    258.54          55.           7.04E-01  บ   
     บ               22.551    .00386           2. ๑      9.      4.1  บ   
     บ             1116.880    243.13          88.           8.87E-01  บ   
     บ               28.389    .00368           2. ๑      7.      1.3  บ   
     บ             1405.570    260.23          53.           6.83E-01  บ   
     บ               26.421    .00357           1. ๑      8.      1.7  บ   
     บ             1308.236    254.59          60.           7.53E-01  บ   
     บ               25.979    .00346           1. ๑      8.      1.3  บ   
     บ             1286.393    253.31          63.           7.69E-01  บ   
     บ               30.828    .00337           1. ๑      7.      1.4  บ   
     บ             1526.194    267.05          48.           6.02E-01  บ   
     บ               26.829    .00332           1. ๑      8.      1.4  บ   
     บ             1328.438    255.77          59.           7.39E-01  บ   
     บ               26.960    .00306           1. ๑      8.      1.8  บ   
     บ             1334.903    256.15          58.           7.34E-01  บ   
     บ               25.382    .00306           1. ๑      8.      3.3  บ   
     บ             1256.871    251.57          66.           7.91E-01  บ   
     บ               25.515    .00285           1. ๑      8.      2.3  บ   
     บ             1263.432    251.96          65.           7.86E-01  บ   
     บ               20.259    .00227           1. ๑     10.       .8  บ   
     บ             1003.542    236.08         108.           9.46E-01  บ   
     บ               26.721    .00222           1. ๑      8.      1.0  บ   
     บ             1323.097    255.46          59.           7.43E-01  บ   
     บ               23.941    .00188           1. ๑      9.      1.1  บ   
     บ             1185.633    247.31          75.           8.41E-01  บ   
     บ               28.791    .00184           1. ๑      7.      2.0  บ   
     บ             1425.434    261.36          51.           6.69E-01  บ   
     บ               24.668    .00180           1. ๑      8.      2.3  บ   
     บ             1221.564    249.47          70.           8.16E-01  บ   
     บ               27.123    .00179           1. ๑      8.      2.0  บ   
     บ             1342.956    256.62          57.           7.28E-01  บ   
     บ               23.422    .00174           1. ๑      9.      1.6  บ   
     บ             1159.950    245.76          80.           8.59E-01  บ   
     บ               28.271    .00170           1. ๑      7.      1.1  บ   
     บ             1399.752    259.89          53.           6.87E-01  บ   
     บ               24.922    .00167           1. ๑      8.      2.5  บ   
     บ             1234.143    250.22          68.           8.07E-01  บ   
     บ               24.403    .00154           1. ๑      8.      2.2  บ   
     บ             1208.460    248.68          72.           8.26E-01  บ   
     บ               31.532    .00141           1. ๑      7.      2.9  บ   
     บ             1560.979    268.98          45.           5.81E-01  บ   
     บ               24.234    .00122           1. ๑      9.      1.8  บ   
     บ             1200.104    248.18          73.           8.31E-01  บ   
     บ               31.396    .00111           0. ๑      7.      3.0  บ   
     บ             1554.268    268.61          46.           5.85E-01  บ   
     บ               26.429    .00105           0. ๑      8.      1.7  บ   
     บ             1308.649    254.62          60.           7.53E-01  บ   
     บ               31.279    .00102           0. ๑      7.      2.7  บ   
     บ             1548.450    268.29          46.           5.88E-01  บ   
     บ               25.612    .00101           0. ๑      8.      1.5  บ   
     บ             1268.245    252.24          64.           7.83E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
